pkg/config/templates/rancherd-13-monitoring.yaml

```diff
@@ -10,6 +10,18 @@ bootstrapResources:
     version: {{ .MonitoringChartVersion }}
     defaultNamespace: cattle-monitoring-system
     repoName: harvester-charts
+    diff:
```

Comment:
@ibrokethecloud  expect the code wondering if we have the diff usage doc anywhere from the Rancher side?

Reply:
probably referred to https://fleet.rancher.io/troubleshooting/#fleet-deployment-stuck-in-modified-state

Reply:
@ibrokethecloud Good fix, I will do a local validation. thanks.

Reply:
@ibrokethecloud Verified: your fix solves the "modified" state in newly installed cluster. Thanks.

Reply:
yep @guangbochen that is the correct document.